spyder/utils/misc.py

```diff
@@ -26,7 +27,7 @@ def __remove_pyc_pyo(fname):
     if osp.splitext(fname)[1] == '.py':
         for ending in ('c', 'o'):
             if osp.exists(fname + ending):
-                os.remove(fname + ending)
+                QFile.moveToTrash(fname + ending)
```

Comment:
```suggestion
                os.remove(fname + ending)
```

Let's revert this change because I don't think it's important to move pyc and pyo files to the trash can since they'll recreated by Python on the next import. Plus, they're placed in the `__pycache__` directory in Python 3.